### PR TITLE
Contact-us journey revamp

### PR DESCRIPTION
**Pull request by JamesPRobinson**
### Checklist

* [ ] Have tests been added to cover any changes?
* [ ] Have E2E tests been added to cover any React changes?
* [ ] Have Accessibility tests been added to cover any React changes?

### CODE REVIEW

File: dataworkspace/dataworkspace/tests/core/test_views.py [hunk 1]
```diff
@@ -54,13 +56,13 @@ def test_tech_support_redirect(self):
...
         )
-        self.assertRedirects(response, reverse("technical-support") + "?email=a@b.com")
+        self.assertRedirects(response, f'{reverse("technical-support")}?email=a@b.com')
 
     def test_add_new_dataset_redirect(self):
         response = self._authenticated_post(
             reverse("support"), {"email": "a@b.com", "support_type": "dataset"}
         )
-        self.assertRedirects(response, reverse("request-data:index"))
+        self.assertRedirects(response, f'{reverse("add-dataset-request")}?email=a%40b.com')
```
Comment (ian-leggett) [inline dataworkspace/dataworkspace/tests/core/test_views.py:65]: Should we assert the ASCII value "@" here like in line 59? Does it matter?

**Review by ian-leggett — COMMENTED** [review]
Should the text area fields that are are in “I need data analysis support or advice” and “I need a custom visualisation reviewed“ be required? If you submit an empty value you get no feedback and nothing happens. Also should we update the breadcrumbs in these pages?

File: dataworkspace/dataworkspace/tests/core/test_views.py (hunk 1 above)
Comment (JamesPRobinson) [reply dataworkspace/dataworkspace/tests/core/test_views.py:65]: No think not, I'll change it

**Review by ian-leggett — APPROVED** [review]

File: dataworkspace/dataworkspace/tests/core/test_views.py [hunk 2]
```diff
@@ -824,15 +844,14 @@ def test_survey_source_is_set_correctly_when_entering_form_from_link(self):
...
+                "how_satisfied": "very-satisfied",
                 "trying_to_do": "other",
                 "trying_to_do_other_message": "Hello",
                 "improve_service": "abc",
-                "describe_experience": "def",
             },
         )
         assert response.status_code == 200
-
+        print(response.rendered_content)
```
Comment (ian-leggett) [inline dataworkspace/dataworkspace/tests/core/test_views.py:854]: Can we remove the print statement

**Review by ian-leggett — APPROVED** [review]

**Pull request merged**